pkg/types/machinepools.go

```diff
@@ -50,6 +50,6 @@ type EC2RootVolume struct {
 // LibvirtMachinePoolPlatform stores the configuration for a machine pool
 // installed on libvirt.
 type LibvirtMachinePoolPlatform struct {
-	// QCOWImagePath
-	QCOWImagePath string `json:"qcowImagePath"`
+	BaseImagePool   string `json:"baseImagePool"`
```

Comment:
What's this about?  It doesn't seem to be used anywhere:

```console
$ git log -1 --oneline origin/pr/271
c59a184 *: provide a default OS image for libvirt
$ git grep -i imagepool origin/pr/271
origin/pr/271:installer/pkg/config-generator/generator.go:                      BaseImagePool:   "default",
origin/pr/271:installer/pkg/config-generator/generator.go:                      BaseImagePool:   "default",
origin/pr/271:pkg/types/machinepools.go:        BaseImagePool   string `json:"baseImagePool"`
```

Reply:
> I figured the actuators may end up using it, so I left it in.

I'm in favor of dropping things until folks ask us to put them back in ;).  They may do that, but they're not very likely to ping us with "hey, we don't need that anymore" ;).

Reply:
Talking this over with @abhinavdahiya, this is going to be needed soon. I'm going to leave this and let the MachineAPI team make use of this.